raft/broadcast_tables/experimental/lang.cc

```diff
@@ -26,9 +26,57 @@ bool is_broadcast_table_statement(const sstring& keyspace, const sstring& column
     return keyspace == db::system_keyspace::NAME && column_family == db::system_keyspace::BROADCAST_KV_STORE;
 }
 
-future<> execute(service::raft_group0_client& group0_client, const query& query) {
+::shared_ptr<cql_transport::messages::result_message>
+to_cql_result(raft::broadcast_tables::query_result result) {
```

Comment:
The translation from lower layer to upper layer should happen in the upper layer; the lower layer should not be aware of the upper layer

Reply:
Any includes from `cql3/*` should eventually disappear